src/main/java/uk/gov/hmcts/darts/audio/component/impl/AudioResponseMapperImpl.java

```diff
@@ -10,6 +12,19 @@
 @Component
 public class AudioResponseMapperImpl implements AudioResponseMapper {
 
+    @Override
+    public AddAudioResponse mapToAddAudioResponse(MediaRequestEntity audioRequest) {
+        AddAudioResponse addAudioResponse = new AddAudioResponse();
+        addAudioResponse.setRequestId(audioRequest.getId());
+        addAudioResponse.setCaseId(audioRequest.getHearing().getCourtCase().getCaseNumber());
+        addAudioResponse.setCourthouseName(audioRequest.getHearing().getCourtroom().getCourthouse().getCourthouseName());
+        addAudioResponse.setDefendants(audioRequest.getHearing().getCourtCase().getDefendantStringList());
+        addAudioResponse.setHearingDate(audioRequest.getHearing().getHearingDate());
+        addAudioResponse.setStartTime(audioRequest.getStartTime().toLocalTime().toString());
```

Comment:
Simply going toLocalTime doesn't feel right to me - I think it would be better to return the full offset string date/time.

In general, I think the team needs to question any use of LocalDateTime / LocalTime in the code as the server system clock will be in UTC.
We deal with UTC timestamps in darts-api and currently you can expect to always get a UTC date stored in the database and returned in the darts-api response.
If we wish get a "local" UK time, then I think it would be less confusing for us to use ZonedDateTime like we have done in some places where we needed the local time in darts-api for some metadata files as part of the audio download.